apps/admin-server/src/pages/projects/[project]/authentication/requiredfields.tsx

```diff
@@ -21,6 +21,7 @@ import { PageLayout } from '@/components/ui/page-layout';
 import { Heading } from '@/components/ui/typography';
 import { Separator } from '@/components/ui/separator';
 import { Checkbox } from '@/components/ui/checkbox';
+import {Spacer} from "@openstad-headless/ui/src";
```

Comment:
Zouden we deze dependency niet willen veranderen naar die van @/components . vgm is dat degene waar de admin components in staan. Als er ooit iets veranderd in de packages/index.css (bijv de base rem veranderd) dan kan dit ook effect hebben op de look van de admin